types/provision/pool.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package provision
+
+import (
+	"context"
+	"errors"
+)
+
+var (
+	ErrPoolNotFound      = errors.New("pool does not exist")
+	ErrTooManyPoolsFound = errors.New("too many pools found")
+)
+
+type Pool struct {
+	Name        string `bson:"_id"`
+	Provisioner string
+	Default     bool
+}
+
+type PoolStorage interface {
```

Comment:
@nettoclaudio nice way to design data =)

Reply:
Just following the existing conventions :P